pkg/partition/partition.go

```diff
@@ -90,9 +90,10 @@ func (d *Disk) createPartitionTable() error {
 }
 
 // addPartition is used to add a partition to the partition table.
-// Currently only a single partition can be created. i.e The method can be called only once for a disk
+// Currently only a single partition can be created i.e, The method can be called only once for a disk.
 // TODO: @akhilerm, add method to create partition with given size
 func (d *Disk) addPartition() error {
+	name := "OpenEBS_NDM"
```

Comment:
Can we make this a constant and then use it here and also in the tests. Since we are using in mulitple places a `const` will be better than using the string. That constant can then be used in the tests also.

You can name the constant as `openEbsNDMPartitionName = "OpenEBS_NDM"`

Reply:
ok, I'll update it. :+1:

Reply:
Tests will remain same, right? I've updated.

Reply:
Nope. You can add the constant [here](https://github.com/openebs/node-disk-manager/blob/0baecdcae8929b6ddc6f850ca5c47eb9150af025/pkg/partition/partition.go#L42) and use it in tests also. No need to use hardcoded strings in test.

Reply:
ok